### ISSUE TITLE
Feature request to install certs for all found domains when non-interactive

### ISSUE THREAD

**Issue by steverobbins** [certbot/certbot#5640]
## My operating system is (include version):

```
# cat /etc/*-release
CentOS release 6.7 (Final)
LSB_VERSION=base-4.0-amd64:base-4.0-noarch:core-4.0-amd64:core-4.0-noarch:graphics-4.0-amd64:graphics-4.0-noarch:printing-4.0-amd64:printing-4.0-noarch
CentOS release 6.7 (Final)
CentOS release 6.7 (Final)
```

## I installed Certbot with (certbot-auto, OS package manager, pip, etc):

```
# wget https://dl.eff.org/certbot-auto
```

## I ran this command and it produced this output:

```
# ./certbot-auto --non-interactive --agree-tos --email steve@example.com
...
Creating virtual environment...
Installing Python packages...
Installation succeeded.
Saving debug log to /var/log/letsencrypt/letsencrypt.log
Plugins selected: Authenticator apache, Installer apache
Missing command line flag or config entry for this setting:
Which names would you like to activate HTTPS for?
example.com? www.example.com

(You can set this with the --domains flag)

IMPORTANT NOTES:
 - Your account credentials have been saved in your Certbot
   configuration directory at /etc/letsencrypt. You should make a
   secure backup of this folder now. This configuration directory will
   also contain certificates and private keys obtained by Certbot so
   making regular backups of this folder is ideal.
```

## Certbot's behavior differed from what I expected because:

I used non-interactive mode but it still prompted for me to specify domains (understandable, param was missing).  I think it's good that it throws this error, however there's no flag for "select all" like there is in the interactive version.

I'd like something like `--domains ALL` or `--domains-all`, etc.

I'm writing a script that will be run on about 20 servers to get certificates.  I'd rather not have to specify domains for each.  I could grep apache files and parse them out, but since Certbot is already doing this, there should be an option to get certificates for everything it can find when non-interactive.

For instance, interactive will show

```
Which names would you like to activate HTTPS for?
-------------------------------------------------------------------------------
1: example.com
2: www.example.com
-------------------------------------------------------------------------------
Select the appropriate numbers separated by commas and/or spaces, or leave input
blank to select all options shown (Enter 'c' to cancel): 
```

I want a non-interactive way to "leave input blank."

TL;DR: non-interactive mode needs a way to create certs for all domains founds without the need to specify actual names.

**Comment by pde** [comment]
I think this is a duplicate of #3887?

**Comment by steverobbins** [comment]
Oof.  Over a year old, probably why I missed it in search.  I'll close.
